scss/mixins/_buttons.scss

```diff
@@ -61,10 +61,11 @@
 // scss-docs-end btn-outline-variant-mixin
 
 // scss-docs-start btn-size-mixin
-@mixin button-size($padding-y, $padding-x, $font-size, $border-radius) {
+@mixin button-size($padding-y, $padding-x, $font-size, $line-height, $border-radius) {
```

Comment:
Changing the order of the arguments is a breaking change, this would have to be moved to the end.

Reply:
ok

Reply:
Moved to the end